pgjdbc/pom.xml

```diff
@@ -33,7 +33,7 @@
     <jdbc.specification.version>4.2</jdbc.specification.version>
     <jdbc.specification.version.nodot>42</jdbc.specification.version.nodot>
     <skip.assembly>false</skip.assembly>
-    <checkstyle.version>8.17</checkstyle.version>
+    <checkstyle.version>[8.18,)</checkstyle.version>
```

Comment:
Was that intentional to keep "open" versions?
I think it is better to settle on a specific version to avoid unexpected failures in case future versions would treat rules in a different way.

Reply:
This was recommended by the checkstyle guys. I think I am in agreement with you here. I'll change it.

Reply:
I think we should try to make our build reproducible, so we should use fixed versions.

Reply:
+1 to fixed versions.